parent/pom.xml

```diff
@@ -1073,7 +1081,7 @@
       <dependency>
         <groupId>io.github.acm19</groupId>
         <artifactId>aws-request-signing-apache-interceptor</artifactId>
-        <version>2.3.1</version>
+        <version>${version.aws-signing}</version>
```

Comment:
❓ that's just a tidying up a missed version property, but not related, right?

Reply:
Exactly :+1: